test/testdata/deployednettemplates/recipes/fastnet/genesis.json

```diff
@@ -0,0 +1,104 @@
+{
+	"NetworkName": "",
+	"VersionModifier": "",
+	"ConsensusProtocol": "future",
+	"FirstPartKeyRound": 0,
+	"LastPartKeyRound": 30000,
```

Comment:
Is this too short? This would have been about 36 hours for a perf network, but for a long-lived network we want this to be much, much longer unless we're going to go through and renew partkeys ourselves. At 3m rounds it's only about 87 days if our rounds are 2.5s

Reply:
agreed, we should choose a bigger value for this longer-running network. It will take several hours to make all the keys though (depending on how many cores you have when running netgoal)

Reply:
Updated to 300,000 for now, to help facilitate faster testing.